app/.env

```diff
@@ -0,0 +1,2 @@
+SECRET_KEY=sk_test_0hQ5W9JaWZEkDLd6CjH0wl4a
+PUBLISHABLE_KEY=pk_test_oV4RE2n0vxcKmOVfiTpdaFUx
```

Comment:
cool, lets untrack this file by doing a `git rm app/.env` then push that commit.